jupyterlab_gitsync/src/service/file.ts

```diff
@@ -53,7 +57,9 @@ export class File implements IFile {
 
   async save() {
     try{
-      await this.context.save();
+      const text = this.doc.getValue();
+      await this._saveFile(text);
+      this.resolver.addVersion(text, 'base');
     } catch (error) {
       console.warn(error);
```

Comment:
Same here. Try to avoid using console output except for debugging.

Reply:
I'm thinking of displaying the errors somewhere on the UI (maybe clicking the status indicator will show a panel with a log of all service operations, ie. save/sync/etc.), but this will most likely be included in a separate PR.